src/image.cpp

```diff
@@ -100,6 +101,7 @@ constexpr auto registry = std::array{
     // needs to be before bmff because some ftyp files are handled as qt and
     // the rest should fall through to bmff
     Registry{ImageType::qtime, newQTimeInstance, isQTimeType, amRead, amNone, amRead, amNone},
+    Registry{ImageType::mkv,     newMkvInstance, isMkvType,   amRead, amNone, amRead, amNone},
```

Comment:
it looks like the format checker on the CI doesn't like the extra spaces
```suggestion
    Registry{ImageType::mkv, newMkvInstance, isMkvType, amRead, amNone, amRead, amNone},
```

Reply:
yes exactly